filament/src/IndirectLight.cpp

```diff
@@ -73,15 +73,15 @@ IndirectLight::Builder& IndirectLight::Builder::radiance(uint8_t bands, float3 c
     if (bands >= 1) {
         irradiance[0] = sh[0] * 0.282095f;
         if (bands >= 2) {
-            irradiance[1] = sh[1] * 0.325735f;
-            irradiance[2] = sh[2] * 0.325735f;
-            irradiance[3] = sh[3] * 0.325735f;
+            irradiance[1] = sh[1] * -0.325735f;
```

Comment:
We'll have to tell Sceneform they need a new drop. Also, are we sure this time? ^^

Reply:
They need either a new drop, or apply these good values themselves.
Are we sure... mh... can we ever be sure?